client/src/components/DataFiles/DataFilesProjectsList/DataFilesProjectsList.js

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useCallback } from 'react';
+import { Link } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { InfiniteScrollTable, LoadingSpinner, Message } from '_common';
+import './DataFilesProjectsList.scss';
+
+const DataFilesProjectsList = () => {
+  const { error, loading, projects } = useSelector(
+    state => state.projects.listing
+  );
+
+  const infiniteScrollCallback = useCallback(() => {});
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch({
+      type: 'PROJECTS_SHOW_SHARED_WORKSPACES'
+    });
+  }, [dispatch]);
+
+  const columns = [
+    {
+      Header: 'Workspace Title',
+      headerStyle: { textAlign: 'left' },
+      accessor: 'description',
+      Cell: el => (
+        <Link
+          className="data-files-nav-link"
+          to={`/workbench/data/shared/${el.row.original.id}`}
+        >
+          {el.value}
+        </Link>
+      )
+    },
+    {
+      Header: 'Owner',
+      accessor: 'owner',
+      Cell: el => <span>{el.value ? el.value : ''}</span>
```

Comment:
I don't have the solution yet, but we're gonna have a problem listing owners from the tapis system listing here, since the `owner` field is always null in the Projects Listing as it exists now.

Reply:
I'm working on a serializer

Reply:
Made a serializer that looks up any associated metadata and injects the PI's information as the owner field